lib/logic/bloc/stats_bloc/stats_bloc.dart

```diff
@@ -384,24 +385,30 @@ class StatsBloc extends Bloc<StatsEvent, StatsState> {
   }
 
   String _getAverageReadingTimeInYear(List<Book> books) {
-    int readTimeInDays = 0;
+    int readTimeInMilliSeconds = 0;
     int countedBooks = 0;
 
     for (Book book in books) {
+      if (book.readingTime != null) {
+        readTimeInMilliSeconds += book.readingTime!.timeInMilliSeconds;
+        countedBooks += 1;
+        continue;
+      }
       if (book.startDate != null && book.finishDate != null) {
         final startDate = book.startDate!;
         final finishDate = book.finishDate!;
-        final timeDifference = finishDate.difference(startDate).inDays;
+        final timeDifference = finishDate.difference(startDate).inMicroseconds;
```

Comment:
I think this should be inMilliseconds, right?

Reply:
Yes, changed